web/server/entrypoint.go

```diff
@@ -27,18 +26,7 @@ import (
 	rutils "go.viam.com/rdk/utils"
 )
 
-var (
-	// GLoggerCamComp is the global logger-to-file for camera components.
-	GLoggerCamComp *logging.Logger
-	viamDotDir     = filepath.Join(os.Getenv("HOME"), ".viam")
-)
-
-func init() {
-	var err error
-	if GLoggerCamComp, err = logging.NewLogger(); err != nil && !errors.Is(err, logging.UnsupportedError{}) {
-		log.Println("cannot create new logger: ", err)
-	}
-}
+var viamDotDir = filepath.Join(os.Getenv("HOME"), ".viam")
```

Comment:
[nit, opt] os.UserHomeDir() would be better than Getenv("HOME")
But as it needs more error checking and this was already like that before, consider it optional.